aries_cloudagent/protocols/issue_credential/v2_0/routes.py

```diff
@@ -616,7 +619,7 @@ async def credential_exchange_send(request: web.BaseRequest):
         cred_preview = (
             V20CredPreview.deserialize(preview_spec) if preview_spec else None
         )
-        async with context.session() as session:
+        async with profile.session() as session:
             conn_record = await ConnRecord.retrieve_by_id(session, connection_id)
             if not conn_record.is_ready:
```

Comment:
put outside async scope

Reply:
Fixed in this MR https://github.com/globalid/aries-cloudagent-python/pull/20/commits/89fc9e5f5a0f4f3ae82c141932309b08f9c3aced